packages/auth-providers-api/src/dbAuth/__tests__/DbAuthHandler.test.js

```diff
@@ -1512,6 +1512,36 @@ describe('dbAuth', () => {
     })
   })
 
+  describe('graphiqlHeader', () => {
+    it('returns graphiqlHeader immediately when it recognized', async () => {
+      // graphiQL header only called in dev
+      process.env.NODE_ENV = 'development'
```

Comment:
Does the environment reset back to test after this test run?

Reply:
Thank you 👍  When the test ends, I will reset the environment variable as 'test'.